mysql/lib/connect.js

```diff
@@ -121,7 +122,7 @@ module.exports = function(c) {
 					if (err) {
 						callback(err);
 					} else {
-						callback(null, []);
+						callback(null, (opts.passThrough)? records :[]);
```

Comment:
The data returned from streamToTableBatch should be the records that failed to insert into the table. Calling 
callback(null, (opts.passThrough)? records :[]);
means that all records failed